### PR TITLE
Tickets/dm 39632

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: python/lsst/ts/atpneumaticssimulator/command_issued.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        Future which ends as follows:
+
+        * result = None when the command has ended successfully.
+        * salobj.ExpectedError if the command has ended in an error.
+    """
+
+    def __init__(self, name: str) -> None:
+        self.timestamp = utils.current_tai()
+        self.name = name
+        self.ack_timestamp: float = 0.0
```
Comment (tribeiro) [inline python/lsst/ts/atpneumaticssimulator/command_issued.py:56]: Wouldn't it be better to have both `ack_timestamp` and `done_timestamp` as `None` to begin with? I know this makes typing harder but would make the interface cleaner. One thing you could do is have this as "private" attributes and make either a get method or an `attribute` to retrieve float values. 

Something like 

```
def get_ack_timestamp(self) -> float:
    if self._ack_timestamp is not None:
        return self._ack_timestamp
    else:
        raise RuntimeError(...)
```

**Review by tribeiro — APPROVED** [review]
Looks good. I only left a couple minor comments for you to consider before merging. But feel free to ignore them if you think they are not relevant.

File: python/lsst/ts/atpneumaticssimulator/command_issued.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+__all__ = ["CommandIssued"]
+
+import asyncio
+
+from lsst.ts import salobj, utils
+
+
+class CommandIssued:
+    """asyncio future to track the progress of an issued low-level controller
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/atpneumaticssimulator/command_issued.py:30]: Good point. Russell and I discussed this class at length and in the end we overlooked the doc string. I fixed it now.

File: python/lsst/ts/atpneumaticssimulator/command_issued.py (hunk 1 above)
Comment (wvreeven) [reply python/lsst/ts/atpneumaticssimulator/command_issued.py:56]: Actually, the timestamp isn't used yet so setting it to `None` is fine for now.

**Pull request merged**